packages/@react-spectrum/menu/test/MenuTrigger.test.js

```diff
@@ -696,4 +696,137 @@ describe('MenuTrigger', function () {
       expect(buttonRef.current.UNSAFE_getDOMNode()).toBe(getByRole('button'));
     });
   });
+
+  describe('MenuTrigger trigger="longPress" behaviour', function () { 
+    it.each`
+      Name             | Component      | props
+      ${'MenuTrigger'} | ${MenuTrigger} | ${{onOpenChange, trigger: 'longPress'}}
+    `('$Name toggles the menu display on spacebar longPress', async function ({Component, props}) {
+      verifyMenuToggle(Component, props, {}, (button, menu) => {
+        if (menu) {
+          triggerPress(button);
+        } else {
+          act(() => {
+            fireEvent.keyDown(button, {key: ' '});
```

Comment:
I believe fireEvent will already wrap in an act, you should just be able to wrap the runAllTimers in it. Followup to that statement, is this runAllTimers needed? https://github.com/adobe/react-spectrum/blob/aba365292f923fe4e4a03eb9272287992a577ebb/packages/%40react-spectrum/menu/test/MenuTrigger.test.js#L112
Can we get a keyUp event as well, after the timer? that way we are more like real behavior in the browser
These aren't quite testing longPress, we're running all timers, which is the same thing we did for all the previous non-longPress tests. I think we should change it up a bit and only run the timers as far as we need, or we need to verify how far we advanced in time.
We don't need the `it.each` syntax, we only have one line in it.

Same comments for the following tests